### PR TITLE
Config: Define global static bind address

### PR DESCRIPTION
**Pull request by TheCharlatan**
Defines a static bind address that will be used for all listens of a running farcaster instance.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -28,13 +28,17 @@ pub const FARCASTER_TESTNET_ELECTRUM_SERVER: &str = "ssl://blockstream.info:993"
...
+
 #[derive(Deserialize, Serialize, Debug, Clone)]
 #[serde(crate = "serde_crate")]
 pub struct Config {
     /// Farcasterd configuration
     pub farcasterd: Option<FarcasterdConfig>,
     /// Syncer configuration
     pub syncers: Option<SyncersConfig>,
+    /// Bind address
+    pub bind_addr: String,
```
Comment (h4sh3d) [inline src/config.rs:41]: This could go under farcasterd or a new peerd section, what do you think?

**Comment by TheCharlatan** [comment]
> Adding a required field after config got generated will break and force the user to update is config.

That's clear to me, but why does it break if there is no config present?

**Comment by TheCharlatan** [comment]
I'm not too keen on this change anymore. It robs us of a bunch of flexibility. I'll think some more about it and will eventually report back.

**Comment by h4sh3d** [comment]
I propose we take the opportunity here to also change the default port to `7067` which is the decimal repr of "FC". This port number as no entry on iana.org :)

**Comment by h4sh3d** [comment]
I'll rebase and propose a commit for the new default port.

**Comment by TheCharlatan** [comment]
> I'll rebase and propose a commit for the new default port.

Ok, I'll leave my hands off this one.

**Review by h4sh3d — APPROVED** [review]
ACK

Waiting on @TheCharlatan ACK and #792

**Comment by TheCharlatan** [comment]
ACK , just left a small additional commit to relax the log level again.

**Pull request merged**